src/Test.EndToEnd.Baselining/Extensions/SarifLogExtensions.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Microsoft.CodeAnalysis.Sarif;
+
+namespace SarifBaseline.Extensions
+{
+    public enum BaselineFilteringMode
+    {
+        None = 0,
+        ToIncludedArtifacts
+    }
+
+    public static class SarifLogExtensions
+    {
+        public const string BaselineFilteringPropertyName = "BaselineFiltering";
+
+        public static BaselineFilteringMode GetBaselineFilteringMode(this SarifLog log)
+        {
+            if (log == null) { throw new ArgumentNullException("log"); }
+
+            BaselineFilteringMode result;
```

Comment:
>result [](start = 34, length = 6)

Nit: call this `mode` because `result` is so heavily used in SARIF to mean something else.

Reply:
Normally I try to use the convention that a variable which will be returned by a function is called result (to tell a reader which one to pay most attention to) but in this case I agree about the conflict with big-R Result.